packages/astro/src/@types/astro.ts

```diff
@@ -2054,6 +2055,101 @@ export interface AstroUserConfig {
 		 * For a complete overview, and to give feedback on this experimental API, see the [Rerouting RFC](https://github.com/withastro/roadmap/blob/feat/reroute/proposals/0047-rerouting.md).
 		 */
 		rewriting?: boolean;
+
+		/**
+		 * @docs
+		 * @name experimental.env
+		 * @type {object}
+		 * @default `undefined`
+		 * @version 4.10.0
+		 * @description
+		 *
+		 * Enables experimental `astro:env` features .
+		 *
+		 * The `astro:env` API lets you configure a type-safe schema for your environment variables, and indicate whether they should be available on the server or the client. Import and use your defined variables from the appropriate `/client` or `/server` module:
+		 *
+		 * ```js
+		 * ---
+		 * import { PUBLIC_POKEAPI } from "astro:env/client"
+		 * 
+		 * const data = fetch(`${PUBLIC_POKEAPI}/pokemon/squirtle`);
+		 * ---
```

Comment:
```suggestion
		 * ---
		 * import { PUBLIC_APP_ID } from "astro:env/client"
		 * import { PUBLIC_API_URL, getSecret } from "astro:env/server"
		 * const API_TOKEN = getSecret("API_TOKEN")
		 * 
		 * const data = await fetch(`${PUBLIC_API_URL}/users`, {
		 * 	method: "POST",
		 * 	headers: {
		 * 		"Content-Type": "application/json",
		 * 		"Authorization": `Bearer ${API_TOKEN}`
		 * 	},
		 * 	body: JSON.stringify({ appId: PUBLIC_APP_ID })
		 * })
		 * ---
```
@sarah11918 this would more realistic and show the usage of every type of variable, wdyt?